content/md/en/docs/tutorials/get-started/monitor.md

```diff
@@ -195,24 +184,24 @@ Alternatively, you can open same [URL](http://localhost:9615/metrics) in a brows
 
 ## Visualizing Prometheus metrics with Grafana
 
-After you have Grafana running, navigate to it in a browser (**the default is https://localhost:3000/**).
-Log in using the default user `admin` and password `admin` and navigate to the data sources page at `localhost:3000/datasources`.
+Once you have Grafana running, navigate to it in a browser (**the default is https://localhost:3000/**).
+Log in (default user `admin` and password `admin`) and navigate to the data sources page at
+`localhost:3000/datasources`.
 
 You then need to select a `Prometheus` data source type and specify where Grafana needs to look for it.
 
-The Prometheus port Grafana needs is NOT the one you set in the `prometheus.yml` file (https://localhost:9615) for where your node is publishing it's data.
+The Prometheus port Grafana needs is NOT the one you set in the `prometheus.yml` file (http://localhost:9615) for where your node is publishing it's data.
 
-With both the Substrate node and Prometheus running, configure Grafana to look for Prometheus on it's default port: https://localhost:9090 (unless you customized it).
+With both the Substrate node and Prometheus running, configure Grafana to look for Prometheus on it's default port: http://localhost:9090 (unless you customized it).
 
 Click `Save & Test` to ensure that you have the data source set correctly. 
 Now you can configure a new dashboard.
 
 ### Template Grafana Dashboard
 
-If you would like a basic dashboard to start [here is a template example](/assets/tutorials/node-metrics/substrate-node-template-metrics.json) that you can `Import`
-in Grafana to get basic information about your node:
+If you would like a basic dashboard to start [here is a template example](../../../../../..//content/media/images/docs/tutorials/06-visualize-node-metrics/substrate-node-template-metrics.json) that you can `Import` in Grafana to get basic information about your node:
```

Comment:
```suggestion
If you would like a basic dashboard to start you can import the following template to get basic information about your node:
```
I don't there's much value in referencing it and then displaying.

Reply:
We need to give this file to them, it's not an image, just how you have this repo structured there is no code assets files yet. It's a stop gap.